pages/sitemap/SitemapHandler.inc.php

```diff
@@ -70,50 +70,72 @@ function _createJournalSitemap() {
 		$issueDao = DAORegistry::getDAO('IssueDAO');
 		$publishedArticleDao = DAORegistry::getDAO('PublishedArticleDAO');
 		$galleyDao = DAORegistry::getDAO('ArticleGalleyDAO');
-		
+
 		$journal = Request::getJournal();
 		$journalId = $journal->getId();
-		
+
 		$doc = XMLCustomWriter::createDocument();
 		$root = XMLCustomWriter::createElement($doc, 'urlset');
 		XMLCustomWriter::setAttribute($root, 'xmlns', SITEMAP_XSD_URL);
-		
+
 		// Journal home
 		XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(),'index','index')));
-		// About page
-		XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(), 'about')));
-		XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(), 'about', 'editorialTeam')));
-		XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(), 'about', 'editorialPolicies')));
+		// User register
+		if ($journal->getSetting('disableUserReg') != 1) {
+			XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(), 'user', 'register')));
+		}
+		// User login
+		XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(), 'login')));
+		// Announcements (the URL for each announcement is not considered)
+		if ($journal->getSetting('enableAnnouncements') == 1) {
+			XMLCustomWriter::appendChild($root, $this->_createUrlTree($doc, Request::url($journal->getPath(), 'announcement')));
+		}
```

Comment:
I think we should include the URL for each announcement. This can be valuable for SEO.